src/views/LoanDetails/LoanDetails.js

```diff
@@ -164,22 +164,34 @@ class LoanDetails extends React.Component {
     const { stripes, resources } = this.props;
     const records = resources?.loanAccountsActions?.records ?? [];
     const total = records.reduce((acc, { amount }) => (acc + parseFloat(amount)), 0);
+    const suspendedAction = records.filter(a => a?.paymentStatus?.name === 'Suspended claim returned') || [];
+    const suspendedStatus = (suspendedAction.length > 0) ? 'Suspended' : '';
 
     if (total === 0) return '-';
 
     const value = parseFloat(total).toFixed(2);
 
     return stripes.hasPerm('ui-users.accounts')
       ? (
-        <button
-          className={css.feefineButton}
-          onClick={(e) => this.feefinedetails(e)}
-          type="button"
-        >
-          {value}
-        </button>
+        <>
+          <button
+            className={css.feefineButton}
+            onClick={(e) => this.feefinedetails(e)}
+            type="button"
+          >
+            {value}
+          </button>
+          <br />
+          {suspendedStatus}
+        </>
       )
-      : value;
+      : (
+        <>
+          {value}
+          <br />
+          {suspendedStatus}
+        </>
+      );
```

Comment:
I'm not crazy about duplicating `{suspendedStatus}` on both sides of this ternary. Maybe lift that logic up a level you have something like 
```
const valueDisplay = stripes.hasPerm('ui-users.accounts') ? <button ...>{value}</button> : value;
const suspendedStatusDisplay = suspendedStatus ? `<br />{suspendedStatus};
return <>{valueDisplay}{suspendedStatusDisplay}</>
```
This way, you don't have to replicate any of your display logic across multiple conditions.

Reply:
@zburke Yes, you're right, the code was simplified